From 17a52c6d12f04853473daf97cef384669f279622 Mon Sep 17 00:00:00 2001
From: Jan-Erik Rediger <janerik@fnordig.de>
Date: Mon, 30 Jul 2018 22:09:12 +0200
Subject: [PATCH] Add some info about events

---
 templates/community/events.hbs | 67 +++++++++++++++++++++++++++++++++-
 1 file changed, 65 insertions(+), 2 deletions(-)

diff --git a/templates/community/events.hbs b/templates/community/events.hbs
index 9a2eee5..76cc3fc 100644
--- a/templates/community/events.hbs
+++ b/templates/community/events.hbs
@@ -1,7 +1,70 @@
 {{#* inline "page"}}
 
-<section>
-  <h1>Events</h1>
+<header>
+  <div class="container">
+    <h1>Events</h1>
+  </div>
+</header>
+
+<section id="community">
+  <div class='container'>
+    <p>
+    Every year there are multiple Rust events around the world, bringing together the community. Despite being early in the year, we’re excited to be able to highlight several events that are already being organized!
+    </p>
+  </div>
+</section>
+
+<section id="community-events">
+  <div class="container">
+    <header>
+      <h2>Highlights</h2>
+      <div class="highlight highlight-red"></div>
+    </header>
+    <div class="row">
+      <div class="four columns">
+        <img style="max-width: 300px;" src='http://rustconf.com/images/rustconf-hero.png' alt='RustConf logo' />
+        <h3>RustConf</h3>
+        <p>The official conference for Rust's world-wide community.</p>
+        <a href="http://rustconf.com/" class="button button-secondary">Next: RustFest Portland</a>
+      </div>
+      <div class="four columns">
+        <img style="max-width:300px;" src="https://blog.rustfest.eu/assets/past_events/4-2018-paris.jpg" alt="RustFest Paris 2018 group photo"/>
+        <h3>RustFest</h3>
+        <p>The premier Rust conference in Europe, changing places every 6 months.</p>
+        <a href="https://rustfest.eu" class="button button-secondary">Next: RustFest Rome</a>
+      </div>
+      <div class="four columns">
+        <img style="max-width:300px;" src="https://secure.meetupstatic.com/photos/event/e/5/1/9/600_464458649.jpeg" />
+        <h3>Rust Day Mexico</h3>
+        <p>A one-day workshop event for Rustaceans in Mexico City.</p>
+        <a href="http://blog.community.rs/2017/07/23/rustdaymx-2017.html" class="button button-secondary">Making of RustDay Mexico City 2017</a>
+      </div>
+    </div>
+</section>
+
+<section id="community-content">
+  <div class="container">
+    <header>
+      <h2>Contact</h2>
+      <div class="highlight highlight-green"></div>
+    </header>
+    <p>The events team helps events to get better in touch with the rest of the community. We advise and bring you in contact with other organisers. Approach the team if you want to get your meetup featured on the events calendar and subsequently the newsletter.</p>
+    <a href="mailto:community@rust-lang.org" class="button button-secondary">Get in contact</a>
+</section>
+
+<section id="rustbridge">
+  <div class="container">
+    <header>
+      <h2>Projects</h2>
+      <div class="highlight highlight-yellow"></div>
+    </header>
+    <div class="row">
+      <div class="four columns">
+        <h3>foss-events-planer</h3>
+        <p>A tool to build your own flexible event. Provides documentation for general subjects and detailed features to tailor your event to your needs.</p>
+        <a href="https://github.com/rust-community/foss-events-planner" class="button button-secondary">foss-events-planer Repository</a>
+      </div>
+  </div>
 </section>
 
 {{/inline}}
-- 
2.18.0

